### ISSUE TITLE
Get sound back onto entrance doorbot

### ISSUE THREAD

**Issue by goatchurchprime** [DoESLiverpool/somebody-should#1326]
We should add an amplifier and speaker to the board in the magnetic lock box by the front door so we get sounds again.  

I think the entrance sounds should be played into the room as people make their entrance, rather than go to the hassle to use the spare wires to play it back at the person who has just touched in from outside.  Therefore this ought to be easy if we get the parts and wire them in.  

I checked the  doorbot hardware by loading the broken kicad files and got the code for the amp as MAX98357A.  https://github.com/DoESLiverpool/doorbot-setup/tree/master/hardware

Does it look like one of these:  https://shop.pimoroni.com/products/adafruit-i2s-3w-class-d-amplifier-breakout-max98357a

**Comment by amcewen** [comment]
Yes, see https://github.com/DoESLiverpool/doorbot-setup/issues/17 for when it was added to the circuit.  I wonder if the doorbot in the Events Room has one on-board already?

**Comment by amcewen** [comment]
The hardware side of this is done, and if you listen carefully you can hear the entrance music.

However, because that's running the desktop version of the Raspberry Pi OS, the audio subsystem is more complicated than I'd expect; and I've not worked out how to set the desktop volume level in a way that doesn't get overridden by something to make it quieter again after a while 😕
